### PR TITLE
CLDR-17669 add Organization.airbnb

### PR DESCRIPTION
**Pull request by srl295**
- Locales.txt

CLDR-17669

- [ ] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

**Review by macchiati — APPROVED** [review]

**Comment by srl295** [comment]
>  Caused by: java.lang.IllegalArgumentException: Cannot have default content locale in Locales.txt: Airbnb ; sr_Cyrl ; modern ; T3 Serbian

versus

> Error:  (TestCLDRLocaleCoverage.java:395)  Error: : airbnb; locale=sr_Cyrl_ME; level=modern; parent=sr_Cyrl; level=null: expected true

**Comment by srl295** [comment]
Test case was wrong, besides being confusing.  take a look

File: tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestCLDRLocaleCoverage.java [hunk 1]
```diff
@@ -376,33 +376,34 @@ private void checkDisjoint(
...
-                                organization
-                                        + "; locale="
-                                        + originalLevel
-                                        + "; level="
-                                        + level
-                                        + "; parent="
-                                        + parent
-                                        + "; level="
-                                        + parentLevel,
-                                parentLevel != null && parentLevel.compareTo(level) >= 0);
```
Comment (srl295) [inline tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestCLDRLocaleCoverage.java:405]: two-for-one test condition

File: tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestCLDRLocaleCoverage.java [hunk 2]
```diff
@@ -393,16 +393,24 @@ public void TestParentCoverage() {
                     if (parent == null || parent.equals(LocaleNames.ROOT)) {
                         break;
                     }
-                    if (!defaultContentLocales.contains(parent) && !parent.equals("en_001")) { // en_001 is generated later from en_GB
```
Comment (macchiati) [inline tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestCLDRLocaleCoverage.java]: The changes in formatting make it hard to see what the **real** change was. How did the original of this file escape being formatted by spotless?

**Review by macchiati — APPROVED** [review]

**Review by AEApple — APPROVED** [review]

File: tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestCLDRLocaleCoverage.java (hunk 2 above)
Comment (srl295) [reply tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestCLDRLocaleCoverage.java]: it didn't escape, it was formatted 'correctly'

The main change is just adding a defaultContent check. The rest is just test output changes.

**Review by AEApple — APPROVED** [review]

File: tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/Locales.txt [hunk 3]
```diff
@@ -38,6 +38,68 @@
...
+Airbnb ; sk ; modern ; T3 Slovak
+Airbnb ; sl ; modern ; T3 Slovenian
+Airbnb ; sq ; modern ; T3 Albanian
+Airbnb ; sr ; modern ; T3 Serbian
+Airbnb ; sr_Cyrl_ME ; modern ; T3 Serbian (Montenegro)
+Airbnb ; sv ; modern ; T2 Swedish
+Airbnb ; sw ; modern ; T3 Swahili
+Airbnb ; th ; modern ; T3 Thai
+Airbnb ; fil ; modern ; T3 Filipino
+Airbnb ; tr ; modern ; T2 Turkish
```
Comment (AEApple) [inline tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/Locales.txt:95]: @srl295 - Can we reorder fil so it's alphabetical?

Comment (srl295) [reply tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/Locales.txt:95]: ah, tl -> fil

yes – want a new PR for it? i'm not sure all of the others are alphabetical though. some might be by tiers

Comment (AEApple) [reply tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/Locales.txt:95]: > ah, tl -> fil
> 
> yes – want a new PR for it? i'm not sure all of the others are alphabetical though. some might be by tiers

yes - sgtm! In the past it was by tier. I've been meaning to reorganize the others, but it's been low priority. I've just been adding things alphabetically for anything new.

**Comment by DavidLRowe** [comment]
Might we want to put CLDR first (by tiers), then other orgs alphabetically? I guess the main goal is to be able to find things easily. Definitely would be nice to have locales listed alphabetically.

**Comment by AEApple** [comment]
> Might we want to put CLDR first (by tiers), then other orgs alphabetically? I guess the main goal is to be able to find things easily. Definitely would be nice to have locales listed alphabetically.

There's a different ticket for that. It's just not high priority. I can reassign it to you if you'd like to take it on?

**Comment by macchiati** [comment]
I think that ticket is for a tool to resort and reformat; once the tool is
in place, makes it easy to caconicalize

**Pull request merged**